libraries/AP_HAL_ChibiOS/UARTDriver.h

```diff
@@ -180,6 +180,7 @@ class ChibiOS::UARTDriver : public AP_HAL::UARTDriver {
 #endif
     ByteBuffer _readbuf{0};
     ByteBuffer _writebuf{0};
+    ByteBuffer _frame_readbuf{0};
```

Comment:
is the intention that _frame_readbuf holds max 1 frame? Or multiple frames?
I was expecting this to be a buffer of frames, so multiple can be outstanding. We could make it just 1 though, in which case why make it a ByteBuffer?

Reply:
Fixed to use A/B buffers